doc/tutorials/dune-package-management/concepts.md

```diff
@@ -0,0 +1,53 @@
+# Concepts
+
+## Motivation
+
+As the OCaml ecosystem is growing, installing dependencies has become more and
+more common. As such the need for package management arose and around 2012
+[opam](https://opam.ocaml.org/) was created. `opam` enabled the creation of a
+central repository of packages called `opam-repository` which contains all
+published OCaml packages. This repository has over the years grown to about
+5000 distinct packages today.
```

Comment:
I think it is a nice idea, I didn't add a link because I didn't know what the "canonical" source for opam-repository was. I think https://ocaml.org/packages is good, but the section is gone. If I add something like this I'll make sure to add that link :+1: